samples/My.Hr/My.Hr.Api/Startup.cs

```diff
@@ -27,7 +27,7 @@ public void ConfigureServices(IServiceCollection services)
             .AddEventPublisher()
             .AddAzureServiceBusSender()
             .AddAzureServiceBusPurger()
-            .AddAzureServiceBusClient(connectionName: nameof(HrSettings.ServiceBusConnection))
+            .AddAzureServiceBusClient(connectionName: nameof(HrSettings.ServiceBusConnection), configure: (o, sp) => o.RetryOptions.MaxRetries = 3)
```

Comment:
I thought service bus client has retries built in?

Reply:
It does, i am demonstrating/testing overriding as a means to verify that extension method.